File: BIT/Power of Two/powerOfTwo.js
const isPowerOfTwo = number => {
  if (number === 0) {
    return 1;
  }
  number & (number - 1) === 0;
}

const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Please input a number: ', (answer) => {
  console.log(isPowerOfTwo(answer) === 0);
  rl.close();
});

File: BIT/Power of Two/powerOfTwo.cpp
#include <stdio.h>
int number;
int isPowerOfTwo(int number) {
	if ((number - 1) & number) {
		return 0;
	} else {
		return 1;
	}
}
int main() {
	scanf("%d", &number);
	printf("%d", isPowerOfTwo(number));
}